### DISCUSSION TITLE
3 mesi scadenza segnalazione

### DISCUSSION THREAD

**Discussion by mippo13** [globaleaks/globaleaks-whistleblowing-software#3882] [Q&A]
Buongiorno, oggi abbiamo presentato la piattaforma ad un seminario. Era presente anche un avvocato che ci ha fatto notare che la normativa parla di 3 mesi per la scadenza della segnalazione entro la quale deve essere presa in carico. La piattaforma invece consente di impostare i giorni (90) che non sempre coincidono con il termine dei 3 mesi.
Volevo chiedervi se si può impostare la piattaforma nel utilizzare i mesi e non i giorni per calcolare la scadenza.
Grazie in anticipo.
Michele

**Comment by evilaliv3** [comment]
Buoongiorno, grazie per la domanda.

La policy di scadenza e le tempistiche di gestione sono due aspetti data ternesi distinti distinti.

Avete tempo 3 mesi per dare un riscontro al segnalante ma ovviamente la segnalazione dovrà essere trattata per più tempo e dovra essere mantenuta per non oltre 5 anni dal momento della sua chiusura.

Le tipiche configurazioni della policy di data retention automatica nel contesto anticorruzione europeo è di 1 o due anni, rinnovabile manualmente dai riceventi secondo necessità.

**Comment by mippo13** [comment]
La ringrazio della risposta. 
Cordiali saluti
